medmodels/matching/tests/test_classic_distance_models.py

```diff
@@ -26,21 +26,31 @@ def test_nearest_neighbor(self):
         t_set = pl.DataFrame([[1, 4, 2]], schema=cols, orient="row")
         covs = ["a", "c"]
 
-        # absolute metric
-        expected_abs = pl.DataFrame([[1.0, 3.0, 5.0]], schema=cols, orient="row")
-        result_abs = cdm.nearest_neighbor(t_set, c_set, "absolute", covariates=covs)
-        self.assertTrue(result_abs.equals(expected_abs))
+        # minkowksi metric
+        expected_mink = pl.DataFrame([[1.0, 3.0, 5.0]], schema=cols, orient="row")
+        result_mink = cdm.nearest_neighbor(
+            t_set, c_set, metric="minkowski", covariates=covs
+        )
+        self.assertTrue(result_mink.equals(expected_mink))
 
-        # mahalanobis metric
-        expected_mah = pl.DataFrame([[1.0, 3.0, 5.0]], schema=cols, orient="row")
-        result_mah = cdm.nearest_neighbor(t_set, c_set, "mahalanobis", covariates=covs)
-        self.assertTrue(result_mah.equals(expected_mah))
+        # euclidean metric
+        expected_euc = pl.DataFrame([[1.0, 3.0, 5.0]], schema=cols, orient="row")
```

Comment:
We allow so many distance metrics and algorithms now - should they all be tested?

Reply:
I will just add a comment that we are trying a different metric and a different algorithm to show that it works and that would cover it I'd say

Reply:
@JabobKrauskopf ?